docs/manual/external-tools.tex

```diff
@@ -1004,6 +1005,99 @@
 You can download it from \myurl{https://sourceforge.net/projects/tide/}.
 \end{sloppypar}
 
+\sectionAndLabel{javac}{javac}
+
+This section explain how to use the Checker Framework with \<javac>.  This is
+useful if you do not wish to use the javac wrapper script described in
+Section~\ref{javac-installation} or to help you adapt your build tool.  This
+section assumes you have downloaded the Checker Framework release zip and set
+the environment variable \<CHECKERFRAMEWORK> to point to the unziped directory.
+You can also get all of the jars mentioned in this section from Maven Central:
+
+\begin{itemize}
+\item \<javac.jar>: \url{https://search.maven.org/artifact/com.google.errorprone/javac/9%2B181-r4173-1/jar}
+\item \<jdk8.jar>: \url{https://search.maven.org/artifact/org.checkerframework/jdk8/2.11.0/jar}
+\item \<checker-qual.jar>: \url{https://search.maven.org/artifact/org.checkerframework/checker-qual/2.11.0/jar}
+\item \<checker.jar>: \url{https://search.maven.org/artifact/org.checkerframework/checker/2.11.0/jar}
+\end{itemize}
+
+\subsectionAndLabel{Java 8}{java8-install}
+
+This sections show what arguments to pass to javac to run the Checker Framework using JDK 8.
+
+\begin{Verbatim}
+javac \
+-J-Xbootclasspath/p:$CHECKERFRAMEWORK/checker/dist/javac.jar \
+-Xbootclasspath/p:$CHECKERFRAMEWORK/checker/dist/jdk8.jar \
+-cp $CHECKERFRAMEWORK/checker/dist/checker-qual.jar \
+-processorpath $CHECKERFRAMEWORK/checker/dist/checker.jar \
+-processor org.checkerframework.checker.nullness.NullnessChecker \
+-source 8 -target 8
+\end{Verbatim}
+
+Below is an explanation of each argument.
+\begin{enumerate}
+\item \<-J-Xbootclasspath/p:\$CHECKERFRAMEWORK/checker/dist/javac.jar>: the
+ErrorProne javac.jar, the OpenJDK Java 9 compiler, must be on the JVM
+bootclasspath. The \<-J> indicates that this is a JVM argument rather than a
+compiler argument.
+
+\item \<-Xbootclasspath/p:\$CHECKERFRAMEWORK/checker/dist/jdk8.jar>: jdk8.jar
+must be on the bootclasspath of the compiler.  (This is a compiler arguments, so do not add \<-J>.)
+
+\item \<-cp \$CHECKERFRAMEWORK/checker/dist/checker-qual.jar>: checker-qual.jar
+must be on the compilation classpath.
+
+\item \<-processorpath \$CHECKERFRAMEWORK/checker/dist/checker.jar>:
+checker.jar must be on the processor path.
+
+\item \<-processor org.checkerframework.checker.nullness.NullnessChecker>:
+choose which checker to run by passing its fully qualified name as a processor.
+(Note, using this options means that javac will not search for annotation
+processors, but rather run only those specified here.)
+
+\item \<-source 8 -target 8>: Because using the ErrorProne \<javac.jar> turns
```

Comment:
Also, java.lang.NoSuchFieldError: ANNOTATION_PROCESSOR_MODULE_PATH if you are compiling with source 9+.